packages/plugins/CrossChainBridge/src/SNSAdaptor/index.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { ApplicationEntry } from '@masknet/shared'
+import { base } from '../base'
+import { useState } from 'react'
+import { CrossChainBridgeDialog } from './CrossChainBridgeDialog'
+
+const sns: Plugin.SNSAdaptor.Definition = {
+    ...base,
+    init(signal, context) {},
+    ApplicationEntries: [
+        {
+            RenderEntryComponent({ disabled }) {
```

Comment:
What about the previous MaskBridge entry? Will we remove that?

Reply:
replace MaskBridge as CrossChainBridge

Reply:
ack